docs/user/hosting.rst

```diff
@@ -49,7 +51,13 @@ as we talk about below.
       we offer a CDN as part of our **Pro plan** and above.
       Please contact support@readthedocs.com to discuss how we can enable this for you.
 
-.. _CloudFlare: https://www.cloudflare.com/
+      We bust the cache on the CDN when the following actions happen:
```

Comment:
```suggestion
      We invalidate and refresh the cache on the CDN when the following actions happen:
```

Reply:
:+1: here, "bust" has some technical jargon smell to it.